bin/MessagesBundle_nl.properties

```diff
@@ -115,6 +115,22 @@ Restart = Herstarten
 Drawing = Tekening
 FidoCad = FidoCadJ extenties
 pcbline_width = PCB lijnbreedte:
+
+Theme_management=Theme and color management
```

Comment:
Put all the untranslated resources at the end of the file. They are easier to find for those who would like to work on them.

Reply:
Maybe so, but I'd still like to review the entire exception handling system if you agree. Take a look here: https://github.com/DarwinNE/FidoCadJ/discussions/254